.travis.yml

```diff
@@ -39,6 +39,17 @@ matrix:
       env:
         - JOBCACHE=3
 
+    - name: Lint code with clippy
+      language: rust
+      rust: stable
+      cache: cargo
+      before_script:
+        - rustup component add clippy
+      script:
+        - cargo clippy --all -- -Dwarnings
+      env:
+        - JOBCACHE=3
```

Comment:
This variable must contain a unique number to prevent cache issues. Please pick a number other than the other values used elsewhere in the matrix.